src/pages/settings/AboutPage.js

```diff
@@ -0,0 +1,169 @@
+import React, {useState, useEffect} from 'react';
+import PropTypes from 'prop-types';
+import {View, ScrollView} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import _ from 'lodash';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Navigation from '../../libs/Navigation/Navigation';
+import ROUTES from '../../ROUTES';
+import styles from '../../styles/styles';
+import Text from '../../components/Text';
+import CONST from '../../CONST';
+import {
+    Link, Eye, MoneyBag, Bug, NewWindow,
+} from '../../components/Icon/Expensicons';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import compose from '../../libs/compose';
+import MenuItem from '../../components/MenuItem';
+import Logo from '../../../assets/images/expensify-cash.svg';
+import {version} from '../../../package.json';
+import openURLInNewTab from '../../libs/openURLInNewTab';
+import ONYXKEYS from '../../ONYXKEYS';
+
+
+const propTypes = {
+    // All reports shared with the user
+    reports: PropTypes.shape({
+        reportID: PropTypes.number,
+        reportName: PropTypes.string,
+    }).isRequired,
+
+    ...withLocalizePropTypes,
+};
+
+const defaultProps = {
+
+};
+
+const AboutPage = ({translate, reports}) => {
+    const [reportID, setreportID] = useState(null);
+    useEffect(() => {
```

Comment:
What is `useEffect` doing here?

Reply:
It listens for the change in `reports`.

**Why?**
Because `reports` is being used to extract the `id` for the navigation to the bot chat.

Reply:
It looks like a Hook, which we [haven't yet introduced ](https://github.com/Expensify/Expensify.cash/blob/main/STYLE.md#are-we-allowed-to-use-insert-brand-new-react-feature-why-or-why-not)to the codebase (we try to use HOC in place of this).

In this case though, I don't think it's necessary -- subscribing to an Onyx key will ensure the reports var is updated automatically. Please check existing cases within the codebase to see this.

Reply:
And also it resolves the id before component mount

Reply:
We don't need to introduce this change, as Onyx is an established pattern that we use everywhere within the app without issue. Please follow the existing usages.

Reply:
Should I implement this using `componentDidMount` and `componentDidUpdate `

Reply:
> Should I implement this using componentDidMount and componentDidUpdate
I'm not sure what you are proposing here?

Reply:
My bad, I have removed the hooks. https://github.com/Expensify/Expensify.cash/blob/c171c132e926578f8e47b011cb743317e1c2263e/src/pages/settings/AboutPage.js#L63